web_src/js/features/repo-issue.js

```diff
@@ -617,19 +618,22 @@ export function initRepoIssueTitleEdit() {
       });
     };
 
-    const pullrequest_target_update_url = $(this).data('target-update-url');
+    const pullrequest_target_update_url = $(this).attr('data-target-update-url');
     if ($editInput.val().length === 0 || $editInput.val() === $issueTitle.text()) {
       $editInput.val($issueTitle.text());
       pullrequest_targetbranch_change(pullrequest_target_update_url);
     } else {
-      $.post($(this).data('update-url'), {
+      $.post($(this).attr('data-update-url'), {
```

Comment:
I don't quite understand, what's the benefit to previously?

Reply:
Luckily, I always write documents

https://docs.gitea.io/en-us/guidelines-frontend/


![image](https://user-images.githubusercontent.com/2114189/217800218-0cd7ff07-f4e9-4dba-a0ed-1d67845d0d1b.png)